.github/workflows/check.yml

```diff
@@ -9,7 +9,7 @@ on:
     branches:
       - main
   pull_request:
-    types: [opened, synchronize, reopened]
```

Comment:
Why?

Reply:
I didnt remove that line, I replaced one type with another.

Reply:
As per https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=synchronize#pull_request

The "synchronize" event that you suggest removing ensures we trigger a build when the main branch gets an update which might break the PR. I see no reason for removing that.

> A pull request's head branch was updated. For example, the head branch was updated from the base branch or new commits were pushed to the head branch.